frontend/talentsearch/resources/js/components/skills/SkillsInDetail/SkillsInDetail.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { Meta, Story } from "@storybook/react";
+import { fakeSkills } from "@common/fakeData";
+import Form from "@common/components/form/BasicForm";
+import Submit from "@common/components/form/Submit";
+import SkillsInDetail, { SkillsInDetailProps } from "./SkillsInDetail";
+
+export default {
+  component: SkillsInDetail,
+  title: "SkillsInDetail",
```

Comment:
In the spirit of cleanup :grin: could we change the title to "Skills/Skills In Detail" so that it ends up in the Skills section of Storybook?